Dockerfile

```diff
@@ -53,25 +52,37 @@ RUN echo "Downloading RapidPro ${RAPIDPRO_VERSION} from https://github.com/$RAPI
     tar -xf rapidpro.tar.gz --strip-components=1 && \
     rm rapidpro.tar.gz
 
+# Install Rust, it's required to build poetry on this version of alpine
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+ENV PATH="/root/.cargo/bin:${PATH}"
+
+# Install poetry
+RUN pip install -U pip && pip install -U poetry
+
+# Uninstall Rust
+RUN rustup self uninstall -y
+
 # Build Python virtualenv
-COPY requirements.txt /app/requirements.txt
-RUN LIBRARY_PATH=/lib:/usr/lib /bin/sh -c "/venv/bin/pip install setuptools" \
-    && LIBRARY_PATH=/lib:/usr/lib /bin/sh -c "/venv/bin/pip install -r /app/requirements.txt" \
-    && runDeps="$( \
-      scanelf --needed --nobanner --recursive /venv \
-              | awk '{ gsub(/,/, "\nso:", $2); print "so:" $2 }' \
-              | sort -u \
-              | xargs -r apk info --installed \
-              | sort -u \
-    )" \
-    && apk add --virtual .python-rundeps $runDeps \
-    # TODO should this be in startup.sh?
-    && cd /rapidpro && npm install npm@6.14.11 && npm install \
+RUN python3 -m venv /venv
+ENV PATH="/venv/bin:${PATH}"
+ENV VIRTUAL_ENV="/venv"
+
+# Install configuration related dependencies
+RUN /venv/bin/pip install --upgrade pip && poetry install --no-interaction && poetry add \
```

Comment:
We probably want to add `--no-dev` to `poetry install`, to make sure that we only install production dependancies.